Extensions/ParticleSystem/particleemitterobject.js

```diff
@@ -381,6 +381,24 @@ gdjs.ParticleEmitterObject.prototype.setParticleBlue2 = function(blue){
     }
 };
 
+gdjs.ParticleEmitterObject.prototype.setParticleColor1 = function(rgbColor){
+    var colors = rgbColor.split(";");
+    if ( colors.length < 3 ) return;
+
+    this.setParticleRed1(parseInt(colors[0]));
```

Comment:
It's safer to always use parseInt(..., 10) to explicitly parse the integer in base 10. Otherwise, if the number start with "0" (which can happen if someone write for example `255;090;150`), it will be parsed as octal.
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt for a few explanations :)